code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -943,30 +903,11 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 
-/datum/clockwork_scripture/create_object/mania_motor //Mania Motor: Creates a powerful obelisk that can be used to broadcast messages or open a gateway to any servant or clockwork obelisk.
```

Comment:
Again, there are other routes to balancing this than just removing it.
If it's too strong, there's the immediate choice of increasing the power cost. Maybe try that first?